coriolis/osmorphing/osmount/base.py

```diff
@@ -258,6 +249,28 @@ def _check_mount_fstab_partitions(
 
         return new_mountpoints
 
+    def _get_symlink_target(self, symlink):
+        target = ''
```

Comment:
NIT: whenever writing methods like this which may or may not return a thing, always use `None` as the default value